lms/djangoapps/badges/migrations/0002_assertions_transfer.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+import json
+
+import os
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        from django.core.files.base import ContentFile
+        from xmodule.modulestore.django import modulestore
+        from badges.events import course_complete
+        classes = {}
+        for badge in orm['certificates.BadgeAssertion'].objects.all():
+            if (badge.course_id, badge.mode) not in classes:
+                course = modulestore().get_course(badge.course_id)
+                image_config = orm['certificates.BadgeImageConfiguration'].objects.get(mode=badge.mode)
+                icon = image_config.icon
+                badge_class = orm.BadgeClass(
```

Comment:
I'm not familiar with South migrations.  Whi is it `orm.BadgeClass` and not `orm['badges.BadgeClass']` here?